<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffusionDialog</class>
 <widget class="QDialog" name="DiffusionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anisotropic Diffusion</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>11</y>
     <width>361</width>
     <height>144</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Number of Iterations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="niterSpinBox">
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Time Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="timeStepSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>0.125000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Conductance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="conductanceSpinBox">
        <property name="value">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Diffusion method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="methodComboBox">
        <item>
         <property name="text">
          <string>Anisotropic Diffusion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Curvature Flow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MinMax Curvature Flow</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiffusionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiffusionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
